cfgov/unprocessed/css/molecules/form-field-with-button.less

```diff
@@ -30,12 +30,40 @@
   tags:
     - cfgov-molecules
 */
+
+.inline_info() {
+    margin-bottom: 0;
+    position: relative;
+    top: 5px;
+    float: right;
+}
+
 // TODO: Look into merging email-signup and form-field-with-button styles
 // Form field with button isn't used anywhere else
 .m-form-field-with-button {
     .m-form-field {
         margin-bottom: unit( (@grid_gutter-width / 2) / @base-font-size-px, em );
     }
+
+    // Note: inline info display breakpoints are set based on length of
+    // the default email signup disclaimer ('Privacy Act statement')
+     &_info {
```

Comment:
If we modify the copy, would we need to change the breakpoints?

Reply:
@sebworks We would, although I think the copy is stable at this point. As I understand it, we generally use a `Privacy Act statement` link here, with the exception of a few cases where we substitute a sentence summarizing the statement text. 

We were thinking a future iteration (after more discussion about links vs. summary) might be to hard-code the text and possibly allow users to select the link location (it sounds like there are a couple of variations on the statement).

The breakpoints are specific to the email signup module, though, and the form-field-with-button is ostensibly generic (although currently only used in email signups). That raises questions about how to handle email signup specific code (like these breakpoints, or a change to the disclaimer field).

Reply:
yeah, I don't think the breakpoints should be contingent on the copy. I would try and use flexbox or something else.

Reply:
@sebworks The challenge is reversing the display order of the disclaimer and button, but I think I have it working with flexbox in https://github.com/cfpb/cfgov-refresh/pull/4101/commits/5c52c967862e2ffd57bcbdbd0ae72d744d3f3546.